task/03-date-tasks.js

```diff
@@ -56,7 +60,13 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+   let year = date.getFullYear();
+
+   return (+year % 4 != 0) ? false :
+          (+year % 100 != 0) ? true :
+          (+year % 400 != 0) ? false : true;
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
'0' != 0
and '0' !== 0)"